build/illume.js

```diff
@@ -11,17 +11,15 @@ var _q = require("@standard-library/q");
 var _kefir = require("kefir");
 
 var offsetAbove = function offsetAbove(y, element) {
-  return y > $(element).offset().top;
+  return y > element.offsetTop;
```

Comment:
That looks good to me. It would be cool to have a default `test` script but I don't know if that should run all of the browsers. Could be interesting…

That is the expected output! It returns two streams, one that emits "active" items, and one that emits "inactive" items.

The way that you use it is like:

```javascript
// <div data-name="foo"></div>
// <div data-name="bar"></div>

const regions = illume("name");

regions.active.onValue((name) => {
  $(`div[data-name="${name}"`).addClass("is-active");
});

regions.inactive.onValue((name) => {
  $(`div[data-name="${name}"`).removeClass("is-active");
});
```

The function passed to `onValue` is called whenever a new section becomes active (through scrolling)